doc/user/inputs.rst

```diff
@@ -1,6 +1,418 @@
-*************************
+******
+Inputs
+******
+
+ARMI input files define the initial state of the reactor model and tell ARMI what kind of analysis should be
+performed on it.
+
+.. note:: We have an :doc:`input walkthrough </tutorials/walkthrough_inputs>` tutorial for a quick 
+    overview of the inputs.
+
+There are several input files:
+
+Settings file
+	Contains simulation parameters (like full power, cycle length, and which physics modules to
+  	activate) and all kind of modeling approximation settings (e.g. convergence criteria)
+
+Blueprints file
+	Contains dimensions and composition of the components/blocks/assemblies in your reactor systems, from fuel 
+  	pins to heat exchangers
+ 
+Fuel management file
+	Describes how fuel moves around during a simulation
+
+
+Depending on the type of analysis, there may be additional inputs required. These include things like
+control logic, ex-core models for transients and shielding, etc.
+
+The core map input files can be graphically manipulated with the 
+:py:mod:`Grid editor <armi.utils.gridEditor>`.
+
+
+The Settings Input File
+=======================
+The **settings** input file defines a series of key/value pairs the define various information about the system you are
+modeling as well as which modules to run and various modeling/approximation settings. For example, it includes:
+
+* The case title
+* The reactor power
+* The number of cycles to run
+* Which physics solvers to activate
+* Whether or not to perform a critical control search
+* Whether or not to do tight coupling iterations
+* What neutronics approximations specific to the chosen physics solver to apply
+* Environment settings (paths to external codes)
+* How many CPUs to use on a computer cluster
+
+This file is a YAML file that you can edit manually with a text editor or with the ARMI GUI.
+
+Here is an excerpt from a settings file:
+
+.. literalinclude:: ../../../armi/tests/armiRun.yaml
+    :language: yaml
+    :lines: 3-15
+
+A full listing of settings available in the framework may be found in the :doc:`Table of all global settings </user/inputs>`.
```

Comment:
I think this is trying to link to itself? Regardless, the link doesn't show up when I build.

Reply:
Fixed